### DISCUSSION TITLE
How to add multiple toolbars?

### DISCUSSION THREAD

**Discussion by tekanokhambane** [GrapesJS/grapesjs#5894] [Q&A] [answered]
Is is possible to add a separate toolbar in a component? Instead of having the default grouped toolbar items, I can add an additional toolbar that is independent from the other toolbar.

**Comment by artf** [comment, marked as answer]
You should be able to achieve it with [Canvas Spots](https://grapesjs.com/docs/modules/Canvas.html#canvas-spots)

**Reply by tekanokhambane** [reply]
Wow thanks a lot , thought it would be too complex
